docs/ossm/ossm2-migration/cluster-wide/README.md

```diff
@@ -0,0 +1,384 @@
+# OpenShift Service Mesh 2.6 Cluster wide --> 3 Migration guide
+This guide is for users who are currently running `ClusterWide` OpenShift Service Mesh 2.6 migrating to OpenShift Service Mesh 3.0. You should first read [this document comparing OpenShift Service Mesh 2 vs. OpenShift Service Mesh 3](../../ossm2-vs-ossm3.md) to familiarize yourself with the concepts between the two versions and the differences in managing the workloads and addons.
+
+## Migrating OpenShift Service Mesh 2.6 Cluster wide to OpenShift Service Mesh 3
+
+### Prerequisites
+- you have read [OpenShift Service Mesh 2 vs. OpenShift Service Mesh 3](../../ossm2-vs-ossm3.md)
+- you have completed all the steps from the [pre-migration checklist](../README.md#pre-migration-checklist)
+- you have verified that your 2.6 `ServiceMeshControlPlane` is using `ClusterWide` mode
+- Red Hat OpenShift Service Mesh 3 operator is installed
+- `IstioCNI` is installed
+- `istioctl` is installed
+
+### Procedure
+In this example, we'll be using the [bookinfo demo](https://raw.githubusercontent.com/Maistra/istio/maistra-2.6/samples/bookinfo/platform/kube/bookinfo.yaml) but you can follow these same steps with your own workloads.
+
+#### Plan the migration
+There will be two cluster wide Istio control planes running during the migration process so it's necessary to plan the migration steps in advance in order to avoid possible conflicts between the two control planes.
+
+There are a few conditions which must be verified to ensure a successful migration:
+- both control planes must share the same root certificate
+
+  This can be achieved simply by installing the 3.0 control plane to the same namespace as 2.6 control plane. The migration procedure below shows how to verify the root cert is shared.
```

Comment:
```suggestion
  This can be achieved by installing the 3.0 control plane to the same namespace as 2.6 control plane. The migration procedure below shows how to verify the root cert is shared.
```
Nitpick - good to avoid using "simple" or "simply" in documentation to describe steps.